support/langkit_support.gpr

```diff
@@ -12,6 +12,14 @@ library project Langkit_Support is
    Library_Kind_Param : Library_Kind_Type := external
      ("LIBRARY_TYPE", external ("LANGKIT_SUPPORT_LIBRARY_TYPE", "static"));
 
+   type OS_Kind is ("windows", "unix", "osx");
+   OS : OS_Kind := External ("LANGKIT_SUPPORT_OS", "windows");
+
+   Soversion := External ("LANGKIT_SUPPORT_SOVERSION", "1");
```

Comment:
For consistency inside the project file, could you rename `Soversion` to `SO_Version`, `Adaflags` to `Ada_Flags`, `Ldflags` to `Ld_Flags` and finally `So_Ext` to `SO_Ext`?